pkg/migrations/op_set_notnull_test.go

```diff
@@ -51,10 +51,10 @@ func TestSetNotNull(t *testing.T) {
 					Name: "02_set_not_null",
 					Operations: migrations.Operations{
 						&migrations.OpAlterColumn{
-							Table:   "reviews",
-							Column:  "review",
-							NotNull: ptr(true),
-							Up:      "(SELECT CASE WHEN review IS NULL THEN product || ' is good' ELSE review END)",
+							Table:    "reviews",
+							Column:   "review",
+							Nullable: ptr(true),
```

Comment:
The various uses of the field in this file should be ‘Nullable: ptr(false)’

Reply:
adjusted